Cargo.toml

```diff
@@ -51,7 +51,7 @@ fallible-streaming-iterator = "0.1.9"
 fast-float = { version = "0.2" }
 flate2 = { version = "1", default-features = false }
 futures = "0.3.25"
-hashbrown = { version = "0.14", features = ["rayon", "ahash", "serde"] }
+hashbrown = { version = "=0.14.5", features = ["rayon", "ahash", "serde"] }
```

Comment:
I don't think pinning helps.

The problem is that polars is using the "raw" feature of hashbrown 0.14 without saying so. You were previously getting that feature indirectly via indexmap's dependency on the same hashbrown, but when indexmap 2.6.0 moves to hashbrown 0.15, you're left with your own hashbrown 0.14 without "raw" anymore. So I think what you really need is this:

```diff
-hashbrown = { version = "0.14", features = ["rayon", "ahash", "serde"] }
+hashbrown = { version = "0.14", features = ["rayon", "ahash", "serde", "raw"] }
```

Reply:
Or if it's only used by your "dtype-categorical" feature, as the issue suggests, you could add:

```toml
dtype-categorical = ["hashbrown/raw"]
```

Reply:
Ah... I'm migrating to 0.15 tomorrow anyway so it'll fix itself.